airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -125,6 +126,8 @@ def test_match_expected(self, connector_spec: Optional[ConnectorSpecification],
         """Check that spec call returns a spec equals to expected one"""
         if connector_spec:
             assert actual_connector_spec == connector_spec, "Spec should be equal to the one in spec.yaml or spec.json file"
+        else:
```

Comment:
This should have been part of https://github.com/airbytehq/airbyte/pull/28519 but I'm adding this now while I see it